@xen-orchestra/web/src/components/backups/target/BackupRepositoriesTable.vue

```diff
@@ -0,0 +1,160 @@
+          >
+            <div v-if="column.id === 'checkbox'" v-tooltip="t('coming-soon')">
+              <UiCheckbox disabled accent="brand" :value="row.id" />
+            </div>
+            <UiButtonIcon
+              v-else-if="column.id === 'more'"
+              v-tooltip="t('coming-soon')"
+              icon="fa:ellipsis"
+              accent="brand"
+              disabled
+              size="small"
+            />
+            <UiLink v-if="column.id == 'title'" size="medium" :icon="column.value.icon" href="/#/settings/remotes">
+              {{ column.value.label }}
+            </UiLink>
+          </td>
+        </tr>
+      </template>
+    </VtsDataTable>
+    <UiTopBottomTable :selected-items="0" :total-items="0">
+      <UiTablePagination v-bind="paginationBindings" />
+    </UiTopBottomTable>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { XoBackupRepository } from '@/types/xo/br.type'
+import type { IconName } from '@core/icons'
+import VtsDataTable from '@core/components/data-table/VtsDataTable.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTablePagination from '@core/components/ui/table-pagination/UiTablePagination.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { usePagination } from '@core/composables/pagination.composable'
+import { useTable } from '@core/composables/table.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { noop } from '@vueuse/shared'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { backupRepositoriesTargets } = defineProps<{
+  backupRepositoriesTargets: XoBackupRepository[]
+}>()
+
+const { t } = useI18n()
+
+const searchQuery = ref('')
+
+const filteredBackupRepository = computed(() => {
+  const searchTerm = searchQuery.value.trim().toLocaleLowerCase()
+
+  if (!searchTerm) {
+    return backupRepositoriesTargets
+  }
+
+  return backupRepositoriesTargets.filter(storageRepository =>
+    Object.values(storageRepository).some(value => String(value).toLocaleLowerCase().includes(searchTerm))
+  )
+})
+
+const { visibleColumns, rows } = useTable('backup-repositories', filteredBackupRepository, {
+  rowId: record => record.id,
+  columns: define => [
+    define('checkbox', noop, { label: '', isHideable: false }),
+    define(
+      'title',
+      record => ({
+        label: record.name,
+        id: record.id,
+        icon: brIcon(record),
+      }),
+      { label: t('backup-repository') }
+    ),
+    define('more', noop, { label: '', isHideable: false }),
```

Comment:
```suggestion
    ),
    define('used-space', noop, { label: t('used-space') }),
    define('remaining-space', noop, { label: t('remaining-space') }),
    define('total-capacity', noop, { label: t('total-capacity') }),
    define('more', noop, { label: '', isHideable: false }),
```
Here I don't know what is the best choice .. display empty column ? or just display one column ? 
<img width="1532" height="324" alt="Image" src="https://github.com/user-attachments/assets/a2542e8d-8845-4519-9f92-24d8b320fe0d" />


and in the case of one column 

<img width="1532" height="324" alt="Image" src="https://github.com/user-attachments/assets/46994fb7-bc4e-42ba-8bf9-76311cec6c9c" />

fix the width of `checkbox` and `more` to `0.5rem`